### PR TITLE
Added General Info and resposive to ActiveMembership

### PR DESCRIPTION
**Pull request by MarianLabensky**

### CODE REVIEW

File: src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        footer = {null}>
+    <Form
+     name="basic"
+     onFinish ={handleFinish}
+     form = {form}>
+        <label htmlFor="datepickerOath" className={classes.formLabel}> Дата присяги </label>
+        <Form.Item 
+        className={classes.formField}
+       name="datepickerOath">
+        <DatePicker format = "YYYY-MM-DD"
```
Comment (voroniak) [inline src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx:70]:
```suggestion
        <DatePicker format = "DD-MM-YYYY" 
```
I think we are using this format

File: src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+     name="basic"
+     onFinish ={handleFinish}
+     form = {form}>
+        <label htmlFor="datepickerOath" className={classes.formLabel}> Дата присяги </label>
+        <Form.Item 
+        className={classes.formField}
+       name="datepickerOath">
+        <DatePicker format = "YYYY-MM-DD" 
+        disabledDate={(cur) => disabledDate(cur ,dates.dateEntry)}
+        defaultValue = { dates.dateOath !== "" ? moment(dates.dateOath, 'YYYY-MM-DD') : undefined}
```
Comment (voroniak) [inline src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx:72]:
```suggestion
        defaultValue = { dates.dateOath !== "" ? moment(dates.dateOath, 'DD-MM-YYYY') : undefined}
```

File: src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        />
+        </Form.Item>
+        
+        <label htmlFor="datepickerEnd" className={classes.formLabel}> Дата завершення </label>
+        <Form.Item 
+        className={classes.formField}
+       name="datepickerEnd">
+        <DatePicker format = "YYYY-MM-DD" 
+        disabledDate={(cur) => disabledDate(cur ,dates.dateEntry)}
+        defaultValue = { dates.dateEnd !== "" ? moment(dates.dateEnd, 'YYYY-MM-DD') : undefined}
```
Comment (voroniak) [inline src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx:84]:
```suggestion
        defaultValue = { dates.dateEnd !== "" ? moment(dates.dateEnd, 'DD-MM-YYYY') : undefined}
```

File: src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        className={classes.selectField}
+        placeholder="Дата присяги"
+        />
+        </Form.Item>
+        
+        <label htmlFor="datepickerEnd" className={classes.formLabel}> Дата завершення </label>
+        <Form.Item 
+        className={classes.formField}
+       name="datepickerEnd">
+        <DatePicker format = "YYYY-MM-DD"
```
Comment (voroniak) [inline src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx:82]:
```suggestion
        <DatePicker format = "DD-MM-YYYY" 
```

**Review by voroniak — CHANGES_REQUESTED** [review]

File: src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx (hunk 4 above)
Comment (MarianLabensky) [reply src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx]: Done

File: src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx (hunk 1 above)
Comment (MarianLabensky) [reply src/pages/userPage/ActiveMembership/UserDates/ModalChangeUserDates.tsx]: Done

**Review by voroniak — APPROVED** [review]

**Review by MykhailoKlapchuk — APPROVED** [review]
Good job

**Review by TonyWebber — APPROVED** [review]

**Pull request merged**